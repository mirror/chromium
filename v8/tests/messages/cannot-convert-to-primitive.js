function Foo() { }

Foo.prototype.toString = Foo.prototype.valueOf = function() { return this; }
print(new Foo());