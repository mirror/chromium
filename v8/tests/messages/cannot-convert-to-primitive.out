native runtime.js:294: Uncaught TypeError: Cannot convert object to primitive value
  throw %MakeTypeError('cannot_convert_to_primitive', []);
  ^
